db/views/ecf_duplicates_v02.sql

```diff
@@ -0,0 +1,70 @@
+SELECT
+  participant_identities.user_id AS user_id,
+  participant_identities.external_identifier AS external_identifier,
+  participant_profiles.id,
+  CASE participant_profiles.type WHEN 'ParticipantProfile::Mentor' THEN 'mentor' ELSE 'ect' END AS profile_type,
+  duplicates.count            AS duplicate_profile_count,
+  latest_induction_records.id AS latest_induction_record_id,
+  latest_induction_records.induction_status,
+  latest_induction_records.training_status,
+  latest_induction_records.start_date,
+  latest_induction_records.end_date,
+  latest_induction_records.school_transfer,
+  latest_induction_records.school_id,
+  latest_induction_records.school_name,
+  schedules.schedule_identifier,
+  cohorts.start_year   AS cohort,
+  lead_providers.name  AS provider_name,
+  teacher_profiles.trn AS teacher_profile_trn,
+  teacher_profiles.id  AS teacher_profile_id,
+  COALESCE(declarations.count, 0) AS declaration_count,
+  ROW_NUMBER() OVER (
+               PARTITION BY participant_profiles.participant_identity_id
+               ORDER BY CASE
+               WHEN latest_induction_records.training_status = 'active' AND latest_induction_records.induction_status = 'active' THEN 1
+               WHEN latest_induction_records.training_status = 'active' AND latest_induction_records.induction_status != 'active' THEN 2
+               WHEN latest_induction_records.training_status != 'active' AND latest_induction_records.induction_status = 'active' THEN 3
+               ELSE 4 END
+  ) AS participant_profile_status,
+  FIRST_VALUE(participant_profiles.id) OVER (
```

Comment:
do we need to do this twice, can we get the primary profile and then it's status?